docs/en/observability/add-observability-data.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[add-observability-data]]
+= Send data to {es}
+
+Now that you have provisioned your first deployment of your {es} cluster, 
+you’re ready to ingest your logs, metrics, uptime, and APM data.
+
+To make full use of Elastic Observability, we recommend that you install the observability 
+products in the following order:
+
+1. Install and configure {filebeat} on your servers to collect log events.
++
+{filebeat} allows you ship log data from sources that come
+in the form of files. It monitors the log files or locations that you specify,
+collects log events, and forwards them to {es}. To ease the collection and parsing of
+log formats for common applications such as Apache, MySQL, and Kafka, a number of
+{filebeat-ref}/filebeat-modules.html[modules], also known as
+https://www.elastic.co/integrations?solution=observability[integrations], are available.
```

Comment:
@masci Great point. They are the same thing. To avoid any confusion, it might be best just to link to the modules section in the docs and remove any mention of integrations. In the future, we'll need to address whether to align the docs with the term integrations.